backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -34,22 +33,48 @@
 import java.nio.charset.StandardCharsets;
 import java.io.ByteArrayInputStream;
 import com.google.common.collect.ImmutableList;
+import com.google.blackswan.mock.filesystem.*;
 
 /** Generate list of anomalies based on data in the csv file using average and threshold. */
 public class SimpleAnomalyGenerator implements AnomalyGenerator {
   private static final String DATA_FILE_LOCATION = "/sample-ramen-data.csv";
-  private static final String EXCEPTION_MESSAGE = "Invalid row format.";
-  private static final String COMMA_DELIMITER = ",";
   private static final int THRESHOLD = 13;
   private static final int NUM_POINTS = 5;
+  private static final String DEFAULT_METRIC = "interest";
+  private static final String DEFAULT_DIMENSION = "ramen";
 
   private final ImmutableList<Anomaly> anomalies;
 
+  /** TODO: Let generators take in parameters of dimension/metric names. */
+  public static SimpleAnomalyGenerator createGenerator() {
+    return new SimpleAnomalyGenerator(
+      // For [push] to git, always use LocalFileSystem, as CloudFileSystem will fail 
+      // unit test without access to key.json. 
+      LocalFileSystem.createSystem().getDataAsStream(DEFAULT_METRIC, DEFAULT_DIMENSION),
```

Comment:
when would we use CloudFileSystem instead of LocalFileSystem - when we deploy the app?

Reply:
yes, we should use the `CloudFileSystem` once we deploy the app, since then we can update files in Cloud Storage without redeploying the application with new csv files and it is also better practice compared to deploying a bunch of data files. 
`LocalFileSystem` can be used when uploading to github and for local testing.